File: node/trello/base/read.js
var Read = {
  init: function(obj, configFileName)
  {
    require('./base').init(obj, configFileName);
    Read.obj = obj;
    obj.members = Read.members;
  },

  members: function(value)
  {
    Read.obj.trello.getData(
      Read.obj.baseUrl + '/' + value + '/members', null,
      function(members) {
        if (members) {
          for (var index in members) {
            console.log(
              '%s（%s）', members[index].fullName, members[index].username
            );
          }
        }
      }
    );
  }
};

module.exports = Read;

File: python/trello/base/read.py
#-*- coding:utf-8 -*-
from base import Base

class Read(Base):
  def __init__(self, configFileName):
    super(Read, self).__init__(configFileName)

  def members(self, value):
    members = self.trello.getData(
      '%s/%s/members' % (self.baseUrl, value)
    )
    if members:
      for member in members:
        print(member['fullName'] + '（' + member['username'] + '）')
